lib/path.ts

```diff
@@ -27,13 +27,26 @@ export function normalize(pathString: string): string {
   return pathString && pathString.replace(/\\/g, "/");
 }
 
+/**
+ * Get the root path of the provided path string. If the provided path string is relative (not
+ * rooted), then undefined will be returned.
+ * @param pathString The path to get the root of.
+ */
+export function getRootPath(pathString: string): string | undefined {
+  let result: string | undefined;
+  if (pathString) {
+    result = path.parse(pathString).root || undefined;
```

Comment:
Isn't that undefined already when path is not rooted?

Reply:
No, it returns an empty string.